_includes/features.html

```diff
@@ -0,0 +1,100 @@
+                <p class="second-title"><i class="fas fa-server"></i> &nbsp; Scalable
+                </p>
+                <div>
+                    <p class="feature-text">
+                        <li class="feature-text">Elastic and linear scalability for a growing data and user base</li>
+                        <li class="feature-text">Data distribution and replication for performance and fault tolerance</li>
+                        <li class="feature-text">Multi-datacenter high availability and hot backups</li>
+                    </p>
+                </div>
+            </div>
+            <div class="col-sm-4 portfolio-item feature-item">
+                <p class="second-title"><i class="fas fa-hand-holding-usd"></i> &nbsp; Open source
+                </p>
+                <div>
+                    <p class="feature-text">All functionality is totally free. No need to buy commercial licenses. JanusGraph is fully Open Source under the Apache 2 license.</p>
+                </div>
+            </div>
+            <div class="col-sm-4 portfolio-item feature-item">
+                <p class="second-title"><i class="fas fa-atom"></i> &nbsp; Transactional
+                </p>
+                <div>
+                    <p class="feature-text">JanusGraph is a transactional database that can support thousands of concurrent users executing complex graph traversals in real time. Support for ACID and eventual consistency.</p>
+                </div>
+            </div>
+            <div class="col-sm-4 portfolio-item feature-item">
+                <p class="second-title"><i class="feature-icon fas fa-database"></i> &nbsp; Data Storage
+                </p>
+                <div>
+                    <p class="feature-text">Graph data can be stored in:</p>
+                    <ul class="central-list">
+                        <li>Apache Cassandra</li>
+                        <li>Apache HBase</li>
+                        <li>Google Cloud Bigtable</li>
+                        <li>Oracle BerkeleyDB</li>
+                    </ul>
+                </div>
+            </div>
+            <div class="col-sm-4 portfolio-item feature-item">
+                <p class="second-title"><i class="feature-icon fas fa-search"></i> &nbsp; Search
+                </p>
+                <div>
+                    <p class="feature-text">Secondary indexes are supported via:</p>
+                    <ul class="central-list">
+                        <li>Elasticsearch</li>
+                        <li>Apache Solr</li>
+                        <li>Apache Lucene</li>
+                    </ul>
+
+                </div>
+            </div>
+            <div class="col-sm-4 portfolio-item feature-item">
+                <p class="second-title"><i class="fas fa-chart-pie"></i> &nbsp; Analytics
+                </p>
+                <div>
+                    <p class="feature-text">JanusGraph OLAP integrations include:</p>
```

Comment:
(nitpick) I think _OLAP_ is not an acronym everyone knows. How about keeping _global graph data analytics_ instead?